scripts/travis/setup.sh

```diff
@@ -17,7 +17,6 @@ DATABASES['default']['HOST'] = 'localhost'
 DATABASES['default']['USER'] = 'travis'
 DATABASES['default']['CONN_MAX_AGE'] = 600
 CELERY_ALWAYS_EAGER = True
-CACHE_BACKEND = 'caching.backends.locmem://'
```

Comment:
pretty sure this is an invalid (deprecated/remove) setting that isn't getting used anymore